src/drivers/boards/px4fmu-v4/board_config.h

```diff
@@ -86,12 +86,17 @@
 
 #define GPIO_SPI_CS_FRAM		(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTD|GPIO_PIN10)
 
+#define GPIO_SPI_CS_BMI055_ACC  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTC|GPIO_PIN15)
```

Comment:
Hi Lorenz, David,
I think we got your point, so can make a change proposed by you for BMI055. Is it than needed to change for MPU or other components? Can you do it? Or shall we make a proposal for board_config.h?